### ISSUE TITLE
Dwarf Therapist permission

### ISSUE THREAD

**Issue by MauroMombelli** [Lazy-Newb-Pack/Lazy-Newb-Pack-Linux#23]
Darf Therapist need permission to poke with other process's ram (dwarffortress in the case), this permission must be set (only one time i think) with something similar to 

setcap cap_sys_ptrace=eip /usr/bin/dwarftherapist

of course it need sudo-root permission

**Comment by lethosor** [comment]
This should be done by the `startlnp` script.

**Comment by MauroMombelli** [comment]
When I started it the first time it didn't ask for password (but I launched
from GUI)

**Comment by lethosor** [comment]
Did you run "startlnp" or "PyLNP" to open the launcher? startlnp launches PyLNP but also checks for DT's permissions and prompts to change them if they're not correct.

**Comment by MauroMombelli** [comment]
i run startlnp, but as normal user (doubleclick + execute)

**Comment by lethosor** [comment]
Did you see an xterm window prompting you for a root password? If not, you could try updating the startlnp script with the latest version from this repo: https://github.com/Lazy-Newb-Pack/Lazy-Newb-Pack-Linux/blob/master/pack/startlnp

**Comment by MauroMombelli** [comment]
i didn't, but i fix it manually before using the new script (because of
missing .png complain), so maybe that's the problem :)

2015-11-11 23:57 GMT+01:00 Lethosor notifications@github.com:

**Comment by lethosor** [comment]
Do you have xterm installed?

**Comment by MauroMombelli** [comment]
Yes, but from the script it seems to use konsole (I also have that one)
